openmp/libomptarget/test/lit.cfg

```diff
@@ -136,7 +140,7 @@ else: # Unices
     if config.cuda_libdir:
         config.test_flags += " -Wl,-rpath," + config.cuda_libdir
     if config.libomptarget_current_target.startswith('nvptx'):
-        config.test_flags += " --libomptarget-nvptx-bc-path=" + config.library_dir + '/DeviceRTL'
```

Comment:
I think the leading whitespace needs to stay in case `config.test_flags_clang` is not empty.